src/main/java/at/ac/uibk/dps/cirrina/cirrina/CirrinaRuntime.java

```diff
@@ -161,7 +178,31 @@ public CuratorFramework newCuratorFramework() {
    * @return OpenTelemetry SDK.
    */
   protected OpenTelemetry getOpenTelemetry() {
-    return AutoConfiguredOpenTelemetrySdk.initialize()
-        .getOpenTelemetrySdk();
+    SpanExporter spanExporter = OtlpGrpcSpanExporter.builder().setEndpoint("http://localhost:4317").build();
```

Comment:
As discussed, please using the conventional `final var` instead of the long types, and also don't try to hardcode endpoints here, instead use environment variables or the arguments passed to the application

Reply:
I would prefer to stick with the auto-configured SDK, because it's more conventional, unless there is a strong reason to do it this way